src/promptflow/promptflow/_cli/_pf/_flow.py

```diff
@@ -218,6 +218,9 @@ def add_parser_test_flow(subparsers):
     add_param_multi_modal = lambda parser: parser.add_argument(  # noqa: E731
         "--multi-modal", action="store_true", help=argparse.SUPPRESS
     )
+    add_param_ui = lambda parser: parser.add_argument(  # noqa: E731
+        "--ui", action="store_true", help=argparse.SUPPRESS
```

Comment:
Does this parameter need to be exposed now

Reply:
I think it's not ready for exposing